src/devices/sound/dmadac.h

```diff
@@ -18,9 +18,35 @@ class dmadac_sound_device : public device_t, public device_sound_interface
 public:
 	dmadac_sound_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock = 0);
 
+	void reset();
```

Comment:
You can’t call this `reset()` because that eclipses `device_t::reset()` (the thing that’s automatically called by the framework, and calls `device_reset()` and `device_reset_after_children()` as well as other stuff).

Reply:
Understood. I renamed it to `initialize_state` instead.